supplements/tashas-cauldron-of-everything/archetypes/druid-wildfire.xml

```diff
@@ -28,15 +28,16 @@
 		</rules>
 	</element>
 	<element name="Circle Spells" type="Archetype Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_ARCHETYPE_FEATURE_CIRCLE_OF_WILDFIRE_CIRCLE_SPELLS">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_DRUID_WILDFIRE_CIRCLE_SPELLS</requirements>
 		<description>
 			<p><em>2nd-level Circle of Wildfire feature</em></p>
 			<p>You have formed a bond with a wildfire spirit, a primal being of creation and destruction. Your link with this spirit grants you access to some spells when you reach certain levels in this class, as shown on the Circle of Wildfire Spells table.</p>
 			<p class="indent">Once you gain access to one of these spells, you always have it prepared, and it doesn’t count against the number of spells you can prepare each day. If you gain access to a spell that doesn’t appear on the druid spell list, the spell is nonetheless a druid spell for you.</p>
 			<h5 class="caption">Circle of Wildfire Spells</h5>
 			<table>
-				<thead>
+				<thread>
 					<tr><td>Druid Level</td><td>Spells</td></tr>
-				</thead>
+				</thread>

```

Comment:
`<thread>`?

Reply:
It screws with my head but <thead> is actually correct.  I think it's short for table header.  I never noticed that until, I just compared the two